pkg/sql/regionliveness/prober.go

```diff
@@ -206,7 +203,8 @@ func (l *livenessProber) QueryLiveness(ctx context.Context, txn *kv.Txn) (LiveRe
 // GetTableTimeout gets maximum timeout waiting on a table before issuing
 // liveness queries.
 func (l *livenessProber) GetTableTimeout() (bool, time.Duration) {
```

Comment:
nit: i think `GetTableTimeout` is a strange name. would `GetProbeTimeout` make more sense?